examples/ElmLogo.elm

```diff
@@ -254,14 +267,15 @@ subscriptions : Model -> Sub Msg
 subscriptions model =
     Sub.batch
         ((if model.isDown then
-            [ Mouse.moves MouseMove ]
+            [ onMouseMove (decodeMouse MouseMove) ]
+
           else
             []
          )
-            ++ [ AnimationFrame.diffs Tick
-               , Mouse.downs MouseDown
-               , Mouse.ups (\_ -> MouseUp)
-               , Window.resizes ResizeWindow
+            ++ [ onAnimationFrame Tick
```

Comment:
I think you should be using [`onAnimationFrameDelta`](https://package.elm-lang.org/packages/elm/browser/latest/Browser-Events#onAnimationFrameDelta) here